pkg/kubeadm/kubeadm.go

```diff
@@ -28,7 +29,7 @@ import (
 const cmdKubeadm string = "kubeadm"
 
 var (
-	cmdOptsCerts      = []string{"alpha", "phase", "certs", "selfsign", "--cert-altnames"}
+	cmdOptsCerts      = []string{"alpha", "phase", "certs", "selfsign", "--apiserver-advertise-address", "0.0.0.0", "--cert-altnames"}
```

Comment:
See https://kubernetes.io/docs/admin/kubeadm/ and search for `--apiserver-advertise-address`:

> If not set (or set to 0.0.0.0) then IP for the default interface will be used.

Unfortunately it is a required flag when generating certs - it wasn't previously (the integration tests found this :))